server/pages/index.js

```diff
@@ -379,6 +379,8 @@ function setUpCSP( req, res, next ) {
 			"'report-sample'",
 			"'unsafe-eval'",
 			'stats.wp.com',
+			'https://widgets.wp.com',
+			'*.wordpress.com',
```

Comment:
It seems those urls were accessed when someone loaded calypso on `/log-in` while logged in and navigated to `/`(Reader).

Testing if `*.files.wordpress.com` matches this.

Reply:
Indeed, that's my assumption as well.